gui/quantum.lua

```diff
@@ -11,262 +11,218 @@ local quickfort = reqscript('quickfort')
-        widgets.ResizingPanel{autoarrange_subviews=true, subviews={
-            widgets.CycleHotkeyLabel{
-                view_id='dir',
-                key='CUSTOM_D',
-                options={{label='North', value={y=-1}},
-                         {label='South', value={y=1}},
-                         {label='East', value={x=1}},
-                         {label='West', value={x=-1}}}},
-            widgets.TooltipLabel{
-                text_to_wrap='Set the dump direction of the quantum stop.',
-                show_tooltip=true}}},
-        widgets.ResizingPanel{autoarrange_subviews=true, subviews={
-            widgets.ToggleHotkeyLabel{
-                view_id='refuse',
-                key='CUSTOM_R',
-                label='Allow refuse/corpses',
-                initial_option=false},
-            widgets.TooltipLabel{
-                text_to_wrap='Note that enabling refuse will cause clothes' ..
+            text_pen=COLOR_GREY
+        },
+        widgets.HotkeyLabel{
+            key='CUSTOM_Q',
+            label='Place QSP under cursor',
+            on_activate=self:callback('commit')
+        },
+        widgets.Panel{autoarrange_subviews=true,
+            frame = { w=75, h=5 },
+            resizable = true,
+            subviews={
+                widgets.EditField{
+                    view_id='name',
+                    key='CUSTOM_N',
+                    on_char=self:callback('on_name_char'),
+                    text=''
+                },
+                widgets.TooltipLabel{
+                    text_to_wrap='Give the quantum stockpile a custom name.',
+                    show_tooltip=true
+                }
+            }
+        },
+        widgets.Panel{
+            autoarrange_subviews=true,
+            frame = { w=75, h=5 },
+            resizable = true,
+            subviews={
+                widgets.CycleHotkeyLabel{
+                    view_id='dir',
+                    key='CUSTOM_D',
+                    options={
+                        {label='North', value={y=-1}},
+                        {label='South', value={y=1}},
+                        {label='East', value={x=1}},
+                        {label='West', value={x=-1}}
+                    }
+                },
+                widgets.TooltipLabel{
+                    text_to_wrap='Set the dump direction of the quantum stop.',
+                    show_tooltip=true
+                }
+            }
+        },
+        widgets.Panel{
+            autoarrange_subviews=true,
+            frame = { w=75, h=5 },
+            resizable = true,
+            subviews={
+                widgets.ToggleHotkeyLabel{
+                    view_id='refuse',
+                    key='CUSTOM_R',
+                    label='Allow refuse/corpses',
+                    initial_option=false
+                },
+                widgets.TooltipLabel{
+                    text_to_wrap='Note that enabling refuse will cause clothes' ..
```

Comment:
this behavior was removed from DF in 50.07 (or thereabouts). You can take this option out and just always have refuse/corpses enabled